src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php

```diff
@@ -327,7 +327,7 @@ public function testReduce()
     {
         $crawler = $this->createTestCrawler()->filterXPath('//ul[1]/li');
         $nodes = $crawler->reduce(function ($node, $i) {
-            return $i == 1 ? false : true;
+            return $i != 1;
```

Comment:
But in this case, it didn't use a strict comparison before. Changing this behaviour now could lead to regressions.